lib/src/base-styles/_nx-lists.scss

```diff
@@ -20,7 +20,7 @@
 }
 
 .nx-list__item {
-  &, > .nx-list__link {
+  &, > .nx-list__link, & > .nx-list__btn {
```

Comment:
Maybe a different class name should be chosen that would be common to both of them

Reply:
> Maybe a different class name should be chosen that would be common to both of them

The best name I could think of was `__click-target` since both are within `--clickable`. I'm actually OK with keeping them distinct, especially since I can imagine ways in which the link and button examples could diverge in use or style. But if you feel strongly and can suggest a good name we can change them.